### PR TITLE
fix: Replace 'concat' directive with ~ and divide routes

### PR DESCRIPTION
**Pull request by GrigoriiBerezin**
Also add key to scalafmt.conf to remove redundant spaces after ~

### CODE REVIEW

File: controllers/src/main/scala/cromwell/pipeline/controller/AuthController.scala [hunk 1]
```diff
@@ -20,46 +20,49 @@ class AuthController(authService: AuthService)(implicit executionContext: Execut
...
-              onComplete(authService.signIn(request)) {
-                case Success(Some(authResponse)) => setSuccessAuthRoute(authResponse)
-                case Failure(IncorrectPasswordException(message)) =>
-                  complete(HttpResponse(StatusCodes.Unauthorized, entity = message))
-                case Failure(InactiveUserException(message)) =>
-                  complete(HttpResponse(StatusCodes.Forbidden, entity = message))
-                case _ => complete(StatusCodes.Unauthorized)
-              }
-          }
+  val signIn: Route = path("signIn") {
```
Comment (myazinn) [inline controllers/src/main/scala/cromwell/pipeline/controller/AuthController.scala]: Make these inner routes private.

Comment (GrigoriiBerezin) [reply controllers/src/main/scala/cromwell/pipeline/controller/AuthController.scala]: Done

**Review by myazinn — APPROVED** [review]

File: controllers/src/main/scala/cromwell/pipeline/controller/ProjectConfigurationController.scala [hunk 2]
```diff
@@ -14,35 +14,40 @@ import scala.util.{ Failure, Success }
...
+      onComplete(projectConfigurationService.getById(ProjectId(projectId))) {
+        case Failure(e)                   => complete(StatusCodes.InternalServerError, e.getMessage)
+        case Success(Some(configuration)) => complete(configuration)
+        case Success(None) =>
+          complete(StatusCodes.NotFound, s"There is no configuration with project_id: $projectId")
+      }
+    }
+  }
+
+  val deactivateConfiguration: Route = delete {
```
Comment (egor-ka) [inline controllers/src/main/scala/cromwell/pipeline/controller/ProjectConfigurationController.scala]: should this route also be private?

Comment (GrigoriiBerezin) [reply controllers/src/main/scala/cromwell/pipeline/controller/ProjectConfigurationController.scala]: Done

**Review by iliaonishchenko — APPROVED** [review]
**Cooooool**

**Review by Dmitrii-Shevchenko — APPROVED** [review]

**Pull request merged**